package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "recogito-js",
-  "version": "1.7.5",
+  "version": "1.7.6",
```

Comment:
Thanks for your effort in keeping the version.  Wonder whether we need to keep increasing it, though.

Reply:
Yea, we probably don't need to. I think once we've finished implementing our changes, then we can release the finished version and include it in fuji.